<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVWizard</class>
 <widget class="QWidget" name="CSVWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSVWizard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_stage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,10,10,10,10,10,10,40">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_intro">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string comment="Noun, the introduction." extracomment="Noun, the introduction.">1. Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_separator">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>2. Separators</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_banking">
          <property name="text">
           <string>3. Banking</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_investing">
          <property name="text">
           <string>4. Investing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_lines">
          <property name="text">
           <string>5. Lines</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_finish">
          <property name="text">
           <string comment="Noun, the end." extracomment="Noun, the end.">6. Finish</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
